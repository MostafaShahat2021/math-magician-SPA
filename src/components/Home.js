function Home() {
  return (
    <div className="home-cont">
      <div className="cont">
        <h5>Welcome to our page</h5>
      </div>
      <p>Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA)</p>
      <p>It is a Single Page App (SPA) that allows users to:</p>
      <ul>
        <li>Make simple calculations.</li>
        <li>Read a random math-related quote.</li>
      </ul>
      <br />
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis eum fuga animi quas! Ips
        illo iusto voluptate error pariatur cumque earum neque quo suscipit, sit dicta accu
        harum ullam voluptates ex numquam impedit aliquam rerum.
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis eum fuga animi quas! Ips
        illo iusto voluptate error pariatur cumque earum neque quo suscipit,
      </p>
    </div>
  );
}
export default Home;
