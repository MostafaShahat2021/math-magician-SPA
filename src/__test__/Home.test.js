import { render } from '@testing-library/react';
import Home from '../components/Home';

it('Home component should render', () => {
  const home = render(<Home />);
  expect(home).toMatchSnapshot();
});
