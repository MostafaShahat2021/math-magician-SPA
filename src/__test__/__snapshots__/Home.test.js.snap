// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home-cont"
      >
        <div
          class="cont"
        >
          <h5>
            Welcome to our page
          </h5>
        </div>
        <p>
          Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA)
        </p>
        <p>
          It is a Single Page App (SPA) that allows users to:
        </p>
        <ul>
          <li>
            Make simple calculations.
          </li>
          <li>
            Read a random math-related quote.
          </li>
        </ul>
        <br />
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis eum fuga animi quas! Ips illo iusto voluptate error pariatur cumque earum neque quo suscipit, sit dicta accu harum ullam voluptates ex numquam impedit aliquam rerum. Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis eum fuga animi quas! Ips illo iusto voluptate error pariatur cumque earum neque quo suscipit,
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home-cont"
    >
      <div
        class="cont"
      >
        <h5>
          Welcome to our page
        </h5>
      </div>
      <p>
        Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA)
      </p>
      <p>
        It is a Single Page App (SPA) that allows users to:
      </p>
      <ul>
        <li>
          Make simple calculations.
        </li>
        <li>
          Read a random math-related quote.
        </li>
      </ul>
      <br />
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis eum fuga animi quas! Ips illo iusto voluptate error pariatur cumque earum neque quo suscipit, sit dicta accu harum ullam voluptates ex numquam impedit aliquam rerum. Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis eum fuga animi quas! Ips illo iusto voluptate error pariatur cumque earum neque quo suscipit,
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
